models/checker.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+  var Checker = sequelize.define('Checker', {
+    userid: DataTypes.INTEGER,
```

Comment:
these will be relationships so we dont have to declare them here probably: https://github.com/izelnakri/nycda-blog-application/blob/refactoring/models/comment.js